.github/workflows/update_sdk_version.yml

```diff
@@ -18,14 +18,17 @@ jobs:
 
       - name: create patch branch
         id: patch
+        env:
+          SDK_VERSION: ${{ github.event.inputs.sdk_version }}
+          PATCH_BRANCH: bumb-sdk-${{ github.event.inputs.sdk_version }}
         run: |
-          git config --global user.name "hyp3-tools-bot"
-          git config --global user.email "uaf-asf-apd@alaska.edu"
-          export PATCH_BRANCH="bumb-sdk-${{ github.event.inputs.sdk_version }}"
+          git config user.name "tools-bot"
+          git config user.email "UAF-asf-apd@alaska.edu"
           echo "::set-output name=branch::${PATCH_BRANCH}"
           git checkout -b ${PATCH_BRANCH}
-          grep -lr "hyp3-sdk.git" | xargs sed -i -E "s/v[0-9]+\.[0-9]+\.[0-9]+/${{ github.event.inputs.sdk_version }}/"
-          git commit -am "Bump SDK version to ${{ github.event.inputs.sdk_version }}"
+          grep -lr "hyp3-sdk.git" | xargs sed -i -E "s/v[0-9]+\.[0-9]+\.[0-9]+/${SDK_VERSION}/"
+          sed -i -E "s/hyp3_sdk==[0-9]+\.[0-9]+\.[0-9]+/hyp3_sdk==${SDK_VERSION#v}/" conda-env.yml
```

Comment:
consider `-r` instead of `-E`; the latter is an undocumented alias
https://linux.die.net/man/1/sed
http://blog.dmitryleskov.com/small-hacks/mysterious-gnu-sed-option-e/